htdocs/langs/en_US/eventorganization.lang

```diff
@@ -50,6 +50,7 @@ DateOfRegistration = Date of registration
 ConferenceOrBoothAttendee = Conference Or Booth Attendee
 ApplicantOrVisitor=Applicant or visitor
 Speaker=Speaker
+EventRegistrationAreClosed=Event registration is closed
```

Comment:
it is present in develop but missing in 20.0
https://github.com/Dolibarr/dolibarr/pull/30198